src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home">
+        <img
+          class="logo_image"
+          src="images/logo.png"
+          alt="Moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptopsandcomputers"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
```

Comment:
It's better for readable
```suggestion
        >
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">Apple</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#samsung">Samsung</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#smartphones">Smartphones</a>
          </li>
          
          <li class="nav__item">
            <a
              class="nav__link"
              href="#laptopsandcomputers"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#gadgets">Gadgets</a>
```